CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
```

Comment:
```suggestion
# Copyright (c) 2024 Apple Inc. and the Swift project authors
```

(Same for other files too.)

Reply:
Ah, yeah, last-minute copy-paste.  Will fix. :)